src/Verbs/PackCommand.cs

```diff
@@ -101,8 +106,11 @@ public async static Task<int> RunAndReturn(PackOptions options)
             var version = GetOrSetPackageVersion(packageXml, options.Version);
 
             Console.WriteLine(Resources.Pack_GetPackageName);
+
             // work out what we are going to call the package
-            var packageFileName = GetPackageFileName(options.OutputDirectory, packageXml, version);
+            var packageFileName = !string.IsNullOrWhiteSpace(options.PackageFileName)
+                ? options.PackageFileName
+                : GetPackageFileName(options.OutputDirectory, packageXml, version);
```

Comment:
The GetPackageFilename method returns `return Path.Combine(folder, $"{name}_{version}.zip");` where `folder` is the output folder you can specify with the -o option on the pack command.

Your current changes makes it so if you specify a name you can't specify an output directory 🙂

Reply:
Ahh, good spot. I’ll take that out of the get file name method. 

I think we can check for .zip too but the version number is probably too far as you don’t know what format they may have added it in. Ultimately this is an advanced option for full control over the name. I think it’s ok to say you need to supply it all.

Reply:
Yep WRT the .zip and version part, I totally agree. Would be a nice quality of life feature - especially that we add the file extension for people who may not think of it, but not the most important 😁

Reply:
Fix has now been committed for both the filepath and .zip extension issues 👍